File: _retired/stories/examples/tsx/manage-customer-orders/cancel-order/process-cancellation-refund/process-cancellation-refund-stories.ts
import type { Story } from '../../story-types'

export const ProcessCancellationRefund = {
  story:       'Process Cancellation Refund',
  actor:       'System',
  domainTerms: ['Cancellation', 'Refund', 'Refund Amount', 'Payment Method'],
  evidence:    [],

} as const satisfies Story

File: contexts/stories/examples/py/manage-customer-orders/cancel-order/process-cancellation-refund/process_cancellation_refund_stories.py
"""Story data — regeneratable. Do not add logic or imports."""

from __future__ import annotations

from typing import Final


PROCESS_CANCELLATION_REFUND: Final = {
    "story":        "Process Cancellation Refund",
    "actor":        "System",
    "domain_terms": ("Cancellation", "Refund", "Refund Amount", "Payment Method"),
    "evidence":     (),
}
